lib/shortcode/events-manager.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+/**
+ * Shortcode Template File for Events Manager
+ */
+
+/**
+ * Create our shortcode for Events Manager
+ */
+add_action( 'init', 'sandwich_events_manager', 11 );
+function sandwich_events_manager() {
+
+	// Check if Shortcake exists
+	if ( ! function_exists( 'shortcode_ui_register_for_shortcode' ) ) {
+		return;
+	}
+
+	// Ensure plugin runs only in admin mode. Do not continue if that's not the case.
+	if ( ! is_admin() ) {
+		return;
+	}
+
+	// Dependency check for Events Manager. Do not proceed if not found.
+	if ( ! defined( 'EM_VERSION' ) ) {
+		return;
+	}
+
+	// Register Shortcake UI for Events Manager Events List
+	shortcode_ui_register_for_shortcode(
+		'events_list',
+		array(
+			'label' => __( 'Events Manager - Events List', 'pbsandwich' ),
```

Comment:
Remove the dash in the name of the shortcode since all our other third-party shortcodes do not have any dashes